pkg/adapter/apiserver/events/events.go

```diff
@@ -25,16 +25,7 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 
 	cloudevents "github.com/cloudevents/sdk-go"
-)
-
-const (
-	AddEventType    = "dev.knative.apiserver.resource.add"
-	UpdateEventType = "dev.knative.apiserver.resource.update"
-	DeleteEventType = "dev.knative.apiserver.resource.delete"
-
-	AddEventRefType    = "dev.knative.apiserver.ref.add"
-	UpdateEventRefType = "dev.knative.apiserver.ref.update"
-	DeleteEventRefType = "dev.knative.apiserver.ref.delete"
+	"github.com/knative/eventing/pkg/apis/sources/v1alpha1"
```

Comment:
can you import this as `sourcesv1alpha1` to make the code clear what the v1alpha1 is referring to.

Also I am not sure we want to lock the event type to the knative version.

Reply:
Changed it to sourcesv1alpha1. 

re: also I am not sure we want to lock the event type to the knative version.

you mean not putting the constants in the types.go ?